notebooks/centrality/Centrality.ipynb

```diff
@@ -0,0 +1,771 @@
+    "# the centrality calls are very straight forwards with the graph being the primary argument\n",
+    "# we are using the default argument values for all centrality functions\n",
+    "\n",
+    "def compute_centrality(_graph) :\n",
+    "    # Compute Degree Centrality\n",
+    "    _d = _graph.degree()\n",
+    "        \n",
+    "    # Comnpute the Betweenness Centrality\n",
+    "    _b = cugraph.betweenness_centrality(_graph)\n",
+    "\n",
+    "    # Compute Katz Centrality\n",
+    "    _k = cugraph.katz_centrality(_graph)\n",
+    "    \n",
+    "    # Compute PageRank Centrality\n",
+    "    _p = cugraph.pagerank(_graph)\n",
+    "    \n",
+    "    return _d, _b, _k, _p"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print function\n",
+    "# being lazy and requiring that the dataframe names are not changed versus passing them in\n",
+    "def print_centrality(n):\n",
+    "    dc_top = dc.sort_values(by='degree', ascending=False).head(n).to_pandas()\n",
+    "    bc_top = bc.sort_values(by='betweenness_centrality', ascending=False).head(n).to_pandas()\n",
+    "    katz_top = katz.sort_values(by='katz_centrality', ascending=False).head(n).to_pandas()\n",
+    "    pr_top = pr.sort_values(by='pagerank', ascending=False).head(n).to_pandas()\n",
+    "    \n",
+    "    df1_styler = dc_top.style.set_table_attributes(\"style='display:inline'\").set_caption('Degree').hide_index()\n",
+    "    df2_styler = bc_top.style.set_table_attributes(\"style='display:inline'\").set_caption('Betweenness').hide_index()\n",
+    "    df3_styler = katz_top.style.set_table_attributes(\"style='display:inline'\").set_caption('Katz').hide_index()\n",
+    "    df4_styler = pr_top.style.set_table_attributes(\"style='display:inline'\").set_caption('PageRank').hide_index()\n",
+    "\n",
+    "    display_html(df1_styler._repr_html_()+df2_styler._repr_html_()+df3_styler._repr_html_()+df4_styler._repr_html_(), raw=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define the path to the test data  \n",
+    "datafile='../data/karate-data.csv'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "cuGraph does not do any data reading or writing and is dependent on other tools for that, with cuDF being the preferred solution.   \n",
+    "\n",
+    "The data file contains an edge list, which represents the connection of a vertex to another.  The `source` to `destination` pairs is in what is known as Coordinate Format (COO).  In this test case, the data is just two columns.  However a third, `weight`, column is also possible"
```

Comment:
If this isn't worth mentioning then disregard, but would this be helpful to add?
```
If a weight column is not present, edge weight values internally default to 1.0
```